### PR TITLE
lsattr: added the command to the Linux section

### PR DESCRIPTION
**Pull request by Cvetomird91**
Added the Linux/AIX specific command for viewing file attributes.

### CODE REVIEW

File: pages/linux/lsattr.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# lsattr 
+
+> List file attributes on a Linux file system
+
+- list file attributes recursively in current directory
+
+`lsattr -R `
```
Comment (waldyrious) [inline pages/linux/lsattr.md]: There's an extra space after R

File: pages/linux/lsattr.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- list attributes of all files in directory, including hidden ones
```
Comment (waldyrious) [inline pages/linux/lsattr.md]: "in the current directory", to match above?

File: pages/linux/lsattr.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`lsattr -a`
+
+- list attributes of directories
```
Comment (waldyrious) [inline pages/linux/lsattr.md]: same here.

**Comment by Cvetomird91** [comment]
Those grammatical and stylistic mistages have been fixed and the previous commits have been squashed into one.

**Comment by waldyrious** [comment]
Thanks for editing and squashing, but unless I'm missing something, my last two comments still apply.

**Comment by rubenvereecken** [comment]
Sad to see this file go! I haven't seen you create a new PR @Cvetomird91, is there any way we can convince you to?

**Comment by Cvetomird91** [comment]
I had trouble syncing my repository fork with the upstream when I submitted this request. 

I can make a new PR with lsattr.md shortly, yes.

**Comment by waldyrious** [comment]
For future reference, this PR was followed-up by #666 (by the same author).

**Pull request closed without merging**